scss/variables.scss

```diff
@@ -0,0 +1,21 @@
+@import url("https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Patrick+Hand&family=Titan+One&display=swap");
+
+$body-bg-color: #5d737e;
+$container-bg-color: #000;
+$sub-container-bg-color: #364958;
+$frame-border-color: #87bba2;
+$box-shadow-color: rgba(225, 225, 225, 0.1);
+$smallerFontStyle: "Architects Daughter", cursive;
+$biggerFontStyle: "Titan One", cursive;
+
+$verySmallFont: 14px;
```

Comment:
can we use `rem` as css units, because of its relative nature
you can refer to this [article](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/) to find out more about `rem`